### PR TITLE
Make Organic Hull a useful very small hull

### PR DESCRIPTION
**Pull request by agrrr3**
[forum discussion](https://www.freeorion.org/forum/viewtopic.php?f=6&t=11822&p=104056)
Reinventing Organic Hull so people actually use it
* Cheap, fast, with lots of fuel and versatile
* almost no structure
* 1 internal, 1 external
* normal detection, great fuel efficiency

### CODE REVIEW

File: default/scripting/ship_parts/Bombard/SP_GRV.focs.txt [hunk 1]
```diff
@@ -39,7 +39,7 @@ Part
             ]
 
         EffectsGroup    // monsters use graviton pulse on player planets
-            scope = NumberOf number = 1 condition = And [
+            scope = MaximumNumberOf number = 1 sortkey = LocalCandidate.ID condition = And [
```
Comment (geoffthemedio) [inline default/scripting/ship_parts/Bombard/SP_GRV.focs.txt]: ?

Comment (agrrr3) [reply default/scripting/ship_parts/Bombard/SP_GRV.focs.txt]: jeez, thanks for catching this. kicked that unrelated change out and rebased

File: default/scripting/ship_hulls/organic/SH_ORGANIC.focs.txt [hunk 2]
```diff
@@ -2,19 +2,17 @@ Hull
...
-        Slot type = External position = (0.50, 0.25)
-        Slot type = External position = (0.70, 0.25)
-        Slot type = Internal position = (0.60, 0.60)
+        Slot type = External position = (0.80, 0.50)
+        Slot type = Internal position = (0.60, 0.50)
     ]
-    buildcost = 14 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_HULL_COST_MULTIPLIER]]
-    buildtime = 3
-    tags = [ "ORGANIC_HULL" "PEDIA_HULL_LINE_ORGANIC" "GOOD_FUEL_EFFICIENCY" ]
+    buildCost = 8 * [[FLEET_UPKEEP_MULTIPLICATOR]] * [[SHIP_HULL_COST_MULTIPLIER]]
```
Comment (geoffthemedio) [inline default/scripting/ship_hulls/organic/SH_ORGANIC.focs.txt]: format was changed recently; these need to be lower-case letters

Comment (agrrr3) [reply default/scripting/ship_hulls/organic/SH_ORGANIC.focs.txt]: done

**Pull request merged**